gamutils/src/test/java/com/genexus/gam/utils/test/resources/CryptographicHash.java

```diff
@@ -0,0 +1,38 @@
+package com.genexus.gam.utils.test.resources;
+
+import org.apache.commons.codec.CharEncoding;
+
+import java.io.UnsupportedEncodingException;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+public class CryptographicHash {
+	private MessageDigest alg = null;
+
+	// private HashAlgorithm alg;
+	public CryptographicHash(String algorithm) {
+		if (algorithm == "" || algorithm == "SHA512")
```

Comment:
Equals must be usead instead of == operator
        if (algorithm.isEmpty() || algorithm.equals("SHA512"))

Reply:
Fixed in commit d1198982aeb9dff8add2e4c33cebe110b5a4301d 
Fix Quality issues and Log